julearn/model_selection/utils.py

```diff
@@ -0,0 +1,47 @@
+"""Utility functions for model selection in julearn."""
+
+# Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
+# License: AGPL
+
+from sklearn.model_selection import check_cv as sk_check_cv
+
+from .final_model_cv import _JulearnFinalModelCV
+
+
+def check_cv(cv=5, classifier=False, include_final_model=False):
+    """Check the CV instance and return the proper CV for julearn.
+
+    Parameters
+    ----------
+    cv : int, str or cross-validation generator | None
+        Cross-validation splitting strategy to use for model evaluation.
+
+        Options are:
+
+        * None: defaults to 5-fold
+        * int: the number of folds in a `(Stratified)KFold`
+        * CV Splitter (see scikit-learn documentation on CV)
```

Comment:
Maybe a link here?

Reply:
That's a copy-paste from other docs. It's not easy to link to that scikit-learn section